app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -195,7 +195,9 @@ def update_host_status
         host = Host.find(input[:host][:id])
         status = host.execution_status_object || host.build_execution_status_object
         status.status = exit_status.to_s == "0" ? HostStatus::ExecutionStatus::OK : HostStatus::ExecutionStatus::ERROR
+        status.refresh unless status.id.nil?
```

Comment:
This would probably be more rails-y and should do essentially the same thing
```suggestion
        status.refresh unless status.new_record?
```

Reply:
ok. Make sense. Thanks.